### PR TITLE
FITWD-26: Fixing mobile header search.

### PR DESCRIPTION
**Pull request by gaurav-facet**

### CODE REVIEW

File: web/themes/custom/milken/scss/_header.scss [hunk 1]
```diff
@@ -498,3 +498,21 @@ body.navbar-active  {
     }
   }
 }
+
+
+.mobile--search {
+  display: block;
```
Comment (uditrwt) [inline web/themes/custom/milken/scss/_header.scss:504]: Remove display:block;

File: web/themes/custom/milken/templates/regions/region--header.html.twig [hunk 2]
```diff
@@ -44,6 +44,7 @@
         </div>
 
         <div class="col text-right px-0">
+          <div class="mobile--search">Search</div>
```
Comment (uditrwt) [inline web/themes/custom/milken/templates/regions/region--header.html.twig:47]: Add search icon in place of the search keyword.

File: web/themes/custom/milken/js/milken.ts [hunk 3]
```diff
@@ -19,6 +19,10 @@ document.querySelector("#scroll_to_top").addEventListener("click", (e) => {
     scrollTop();
 });
 
+jQuery('.mobile--search').click(function(){
```
Comment (uditrwt) [inline web/themes/custom/milken/js/milken.ts:22]: Use on('click', function(){}); in place of click.

File: web/themes/custom/milken/scss/_header.scss (hunk 1 above)
Comment (gaurav-facet) [reply web/themes/custom/milken/scss/_header.scss]: Removed

File: web/themes/custom/milken/templates/regions/region--header.html.twig (hunk 2 above)
Comment (gaurav-facet) [reply web/themes/custom/milken/templates/regions/region--header.html.twig]: Added search icon

File: web/themes/custom/milken/js/milken.ts (hunk 3 above)
Comment (gaurav-facet) [reply web/themes/custom/milken/js/milken.ts]: This is fixed.

**Pull request merged**